src/components/fileinput/_fileinput.scss

```diff
@@ -110,4 +117,36 @@
   text-shadow: 0 1px 0 $white;
   opacity: .2;
   cursor: pointer;
+}
+
+.cmp-adaptiveform-fileinput__dragarea {
+  border: 1px dashed #666;
+  border-radius: 5px;
+  padding: 10px 5px;
+}
+
+.cmp-adaptiveform-fileinput__dragtext {
+  padding-bottom: 10px;
+  padding-bottom: 20px;
+}
+
+.cmp-adaptiveform-fileinput__icon {
+  height: 50px;
+  width: 50px;
+  background-size: cover;
+  background-image: url('');
+}
+
+.cmp-adaptiveform-fileinput__filesize {
+  float: left;
```

Comment:
not working for RTL

Reply:
removed will be handled automatically by the parent container